Cannot install version 0.12.0

When running `npm i graphql-codegen-typescript-validation-schema`

I get the following error message:
```
npm ERR! code 1
npm ERR! path [...]/node_modules/graphql-codegen-typescript-validation-schema
npm ERR! command failed
npm ERR! command sh -c -- npx only-allow pnpm
npm ERR! npm ERR! code SELF_SIGNED_CERT_IN_CHAIN
npm ERR! npm ERR! errno SELF_SIGNED_CERT_IN_CHAIN
```

Downgrading to version 0.11.1 fixes the issue.
I believe that line: `"preinstall": "npx only-allow pnpm",` should be used only in projects, not libraries like this one, as it also propagates to projects using it. I think the intention behind this line was to enforce that every contributor is using pnpm, not every end-user

--- a/.eslintrc.json
+++ b/.eslintrc.json
@@ -43,6 +43,8 @@
   "ignorePatterns": [
     "dist",
     "node_modules",
-    "example"
+    "example",
+    "vitest.config.ts",
+    "tsconfig.json"
   ]
 }

--- a/.prettierignore
+++ b/.prettierignore
@@ -5,3 +5,4 @@ package-lock.json
 .DS_Store
 tsconfig.tsbuildinfo
 example
+vitest.config.ts

--- a/package.json
+++ b/package.json
@@ -15,7 +15,6 @@
     "README.md"
   ],
   "scripts": {
-    "preinstall": "npx only-allow pnpm",
     "type-check": "tsc --noEmit",
     "type-check:yup": "tsc --strict --noEmit example/yup/schemas.ts",
     "type-check:zod": "tsc --strict --noEmit example/zod/schemas.ts",

